diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Embers CLI, Ember usually will use much less code and look more elegant than most other
```

Comment:
I'm not sure about Ember using less code or being more elegant than others. 
Mostly I was looking for more concrete things:
Front end routing
Easily display dynamic content using two-way binding
No more AJAX - Ember takes care of making API calls for you.

.5/1